.github/workflows/test-all.yml

```diff
@@ -35,12 +73,51 @@ jobs:
         cp config/ci.config.json config/project.json
         yarn
     - name: yarn build
-      run: yarn build
+      run: yarn build:ci:auth
     - name: Set start timestamp env var
       run: echo "FIREBASE_CI_TEST_START_TIME=$(date +%s)" >> $GITHUB_ENV
     - name: Run unit tests
       run: |
-        xvfb-run yarn test:ci
+        xvfb-run yarn test:ci:auth
+        node scripts/print_test_logs.js
+      env:
+        FIREBASE_TOKEN: ${{ secrets.FIREBASE_CLI_TOKEN }}
+    - name: Generate coverage file
```

Comment:
I'm curious what the coverage step is going to do when it only has a subset of packages to work with but as it apparently currently doesn't work at all https://coveralls.io/github/firebase/firebase-js-sdk?branch=master I don't think it's an immediate concern. Future todo to figure out why it hasn't been working I guess.

Reply:
Yeah, I wasn't quite sure about this either.